worker/appm/conversion/plugin.go

```diff
@@ -169,6 +177,33 @@ func conversionServicePlugin(as *typesv1.AppService, dbmanager db.Manager) ([]v1
 	return initContainers, containers, &bootSequence, nil
 }
 
+func createDataSourceNameEnv(envs *[]v1.EnvVar) v1.EnvVar {
```

Comment:
define in the plugin spec config item.

Reply:
config item is an attribute of mysqld_exporter and is static. The DataSourceName is dynamic, determined according to the environment variables such as `MYSQL_USERNAME`, `MYSQL_PASSWORD` of the component.